mkosi/config.py

```diff
@@ -3701,7 +3708,10 @@ def parse_config_one(self, path: Path, profiles: bool = False, local: bool = Fal
                         )
 
         if path.exists():
-            logging.debug(f"Including configuration file {Path.cwd() / path}")
+            p = Path.cwd() / path
```

Comment:
Should use a different name here otherwise mypy gets angry about types because python for some insane reason doesn't do block scoping

Reply:
Yep. I renamed it to `abs_path`.